### PR TITLE
add carousel-asia

### PR DESCRIPTION
**Pull request by erlandazakaria**

### CODE REVIEW

**Review by marikagentils — APPROVED** [review]

**Review by marikagentils — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # v2.2.0
 ### Add
-- Add [Step List component](https://github.com/AdactiveSAS/adsum-react-components/tree/master/packages/adsum-steplist).
+- Add [Step List component](https://github.com//adsum-react-components/tree/master/packages/adsum-steplist).
```
Comment (marikagentils) [inline CHANGELOG.md]: Don't you need "adactiveasia" between de two " / " ?

File: packages/adsum-carousel-asia/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        renderBottomCenterControls: null,
+        arrows: false,
+        pauseOnHover: false,
+        slidesToShow: 1,
+        slidesToScroll: 1,
+        adaptiveHeight: true,
+        wrapAround: true,
+        autoSlide: false,
+        autoSlideInterval: 10000,
+    }
```
Comment (marikagentils) [inline packages/adsum-carousel-asia/README.md]: In **AdsumCarousel.js** your defaultProps are :
```
static defaultProps = {
        isOpen: false,
        medias: [],
        onMediaTouch: null,
        carouselOptions: {
            dragging: false,
            swiping: false,
            speed: 1000,
            renderCenterLeftControls: null,
            renderCenterRightControls: null,
            renderCenterBottomControls: null,
            renderBottomCenterControls: null,
            arrows: false,
            pauseOnHover: false,
            slidesToShow: 1,
            slidesToScroll: 1,
            adaptiveHeight: true,
            wrapAround: true,
        },
        dynamicAutoPlayInterval: false,
        style: {},
        ButtonModalForImage: null,
        autoSlide: false,
        autoSlideInterval: 10000,
    };
```
Shouldn't it be the same in your README file ?

File: CHANGELOG.md (hunk 1 above)
Comment (erlandazakaria) [reply CHANGELOG.md]: should be adactiveasia between two "/"

File: packages/adsum-carousel-asia/README.md (hunk 2 above)
Comment (erlandazakaria) [reply packages/adsum-carousel-asia/README.md]: oh, I thought I put it already

**Pull request merged**